code/game/objects/machinery/mortar.dm

```diff
@@ -72,9 +72,18 @@
 	// used for keeping track of different mortars and their types for cams
 	var/static/list/id_by_type = list()
 
+	var/list/linked_binoculars = list()
+	var/obj/item/mortar_kit/linked_mortar
+
 /obj/machinery/deployable/mortar/Initialize(mapload, _internal_item, deployer)
 	. = ..()
 
+	RegisterSignal(src, COMSIG_ITEM_UNDEPLOY, PROC_REF(handle_undeploy_references))
+	linked_mortar = get_internal_item()
```

Comment:
Why does the linked_mortar var exist if you can just get_internal_item

Reply:
using internal item from now on